### PR TITLE
feat: add new navdata command explaining the use of navdata

### PR DESCRIPTION
**Pull request by pdellaert**
## Description

Refactor the existing navdata issue as it is specific to navigraph and introduce a new navdata command to highlight the use of navdata with standard navdata, 3rd party navdata or external DB navdata (which FBW doesn't do)

## Test Results

Screen Shot 2022-10-16 at 4 53 52 PM
Screen Shot 2022-10-16 at 4 56 25 PM

## Discord Username
straks#7240

### CODE REVIEW

File: src/commands/support/navdata.ts [hunk 1]
```diff
@@ -2,19 +2,24 @@ import { CommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
 import { makeEmbed, makeLines } from '../../lib/embed';
 
-const NAVDATA_URL = 'https://cdn.discordapp.com/attachments/785976111875751956/912394143844696154/unknown.png';
-
 export const navdata: CommandDefinition = {
-    name: 'navdata',
-    description: 'Provides help with Navigraph navdata reinstall',
+    name: ['navdata'],
```
Comment (benw202) [inline src/commands/support/navdata.ts:6]: Do we need the `[ ]` on commands without an alias? I have seen some files with and without. Is it correct to have all with? If so, I feel a refactor incoming haha

Comment (pdellaert) [reply src/commands/support/navdata.ts:6]: I want to make it uniform and a standard. We can make this part of the split command effort as as we will be touching so many command anyway?

Comment (oim1) [reply src/commands/support/navdata.ts:6]: > Do we need the `[ ]` on commands without an alias? I have seen some files with and without. Is it correct to have all with? If so, I feel a refactor incoming haha

No it is not needed. Both will work fine. It looks cleaner without the brackets on the single element though.

Comment (pdellaert) [reply src/commands/support/navdata.ts:6]: I personally prefer to have a consistent standard, which would be with `[ ]`. The advantage of it, is that it automatically becomes clear that multiple commands/aliases can be added, even when looking at existing simple commands and the code looks the same everywhere.

 If there's general disagreement, I'll remove it.

Comment (NathanInnes) [reply src/commands/support/navdata.ts:6]: Dont see the need, both work just as fine.

Comment (benw202) [reply src/commands/support/navdata.ts:6]: Sounds good, will refactor for uniformity at some point

**Review by benw202 — APPROVED** [review]

**Pull request merged**